lib/Core/Dispatcher/RequestContext.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Phpactor\LanguageServer\Core\Dispatcher;
+
+use Amp\CancellationToken;
+use Phpactor\LanguageServer\Core\Handler\Handlers;
+use Phpactor\LanguageServer\Core\Server\Transmitter\ConnectionMessageTransmitter;
+
+class RequestContext
```

Comment:
Compromise to avoid what would probably be a better solution: booting a new container.